### PR TITLE
MM-T341 Download link on preview - Image file (non SVG)

### PR DESCRIPTION
**Pull request by clarmso**
#### Summary

A download link should be available on the preview. Clicking on the link should fetch the file successfully.

#### Ticket Link

https://automation-test-cases.vercel.app/test/MM-T341

#### Related Pull Requests

- Has server changes N/A
- Has redux changes N/A
- Has mobile changes N/A

#### Screenshots

Screen Shot 2020-08-30 at 11 34 32 AM

### CODE REVIEW

File: e2e/cypress/integration/files_and_attachments/upload_files_spec.js [hunk 1]
```diff
@@ -148,6 +148,29 @@ describe('Upload Files', () => {
         });
     });
 
+    it('MM-T341 Download link on preview - Image file (non SVG)', () => {
+        ['small-image.png', 'image-file.jpg', 'image-file.bmp', 'image-file.gif', 'image-file.tiff'].forEach((image) => {
```
Comment (clarmso) [inline e2e/cypress/integration/files_and_attachments/upload_files_spec.js:152]: `.ico` and `.webp` files don't have image thumbnail preview, but SVG does. Could someone please confirm if SVG is intended to be included in this test?

Comment (srkgupta) [reply e2e/cypress/integration/files_and_attachments/upload_files_spec.js:152]: Thanks for checking @clarmso. The file types you have chosen in sufficient. This test specifically indicates non SVG files. Hence we can ignore the SVG files.

**Review by srkgupta — APPROVED** [review]
Thanks @clarmso. The test looks great and is working fine. Approving the PR.

**Review by saturninoabril — APPROVED** [review]
Thanks @clarmso, LGTM!

**Pull request merged**